RHDEVDOCS-4430 - Loki Log Based Alerting (#57371)
openshift/openshift-docs: snippets/logging-alertingrule-inf-callouts-snip.adoc

.Example infrastructure AlertingRule CR
[source,yaml]
----
  apiVersion: loki.grafana.com/v1
  kind: AlertingRule
  metadata:
    name: loki-operator-alerts
    namespace: openshift-operators-redhat <1>
    labels: <2>
      openshift.io/cluster-monitoring: "true"
  spec:
    tenantID: "infrastructure" <3>
    groups:
      - name: LokiOperatorHighReconciliationError
        rules:
          - alert: HighPercentageError
            expr: | <4>
              sum(rate({kubernetes_namespace_name="openshift-operators-redhat", kubernetes_pod_name=~"loki-operator-controller-manager.*"} |= "error" [1m])) by (job)
                /
              sum(rate({kubernetes_namespace_name="openshift-operators-redhat", kubernetes_pod_name=~"loki-operator-controller-manager.*"}[1m])) by (job)
                > 0.01
            for: 10s
            labels:
              severity: critical <5>
            annotations:
              summary: High Loki Operator Reconciliation Errors <6>
              description: High Loki Operator Reconciliation Errors <7>
----

> <1> The `namespace` where this AlertingRule is created needs to have a label matching the LokiStack `spec.rules.namespaceSelector` definition.
> <2> The labels block needs to match the LokiStack `spec.rules.selector` definition.

Reviewer: ```suggestion
<1> The `namespace` where this AlertingRule is created must have a label matching the LokiStack `spec.rules.namespaceSelector` definition.
<2> The `labels` block must match the LokiStack `spec.rules.selector` definition.
```
